client/blocks/reader-post-card/conversation-post.jsx

```diff
@@ -13,7 +13,11 @@ class ConversationPost extends Component {
 		return (
 			<div className="reader-post-card__conversation-post">
 				<CompactPostCard { ...this.props } />
-				<ConversationPostList post={ this.props.post } commentIds={ this.props.commentIds } />
+				<ConversationPostList
+					post={ this.props.post }
+					commentIds={ this.props.commentIds }
+					filterParents={ false }
```

Comment:
this seems to be the only place that ConversationPostList is used, so I'd probably just make `filterParents=false` the default and not add the property